content/get-involved.ts

```diff
@@ -114,6 +122,64 @@ const getInvolvedContent: GetInvolvedContent = {
       },
     ],
   },
+  testimonies: [
+    {
+      id: 1,
+      testimonial: 'What initially drew me to Clearviction was their unwavering commitment to making a positive impact on the lives of people with convictions. But what has made me want to stay and contribute long-term is the team and the environment we work in.',
+      imageSrc: '/illustrations/gabriel.png',
+      name: 'GABRIEL EGER',
+      cvrole: 'Software Engineer',
+
```

Comment:
If we add pronouns can use this line, otherwise delete extra lines in all of these objects.

Reply:
I think we should put pronouns

Reply:
I will ask them

Reply:
pronouns will not be added extra lines will be removed.